std_msgs/Header header

uint32 alpha_agent_id
string formation_type
geometry_msgs/Point[] target_positions
float32 formation_spacing
float32 movement_speed
bool maintain_formation
string formation_objective
